pom.xml

```diff
@@ -60,6 +60,10 @@
             <artifactId>spring-boot-starter-test</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-data-jpa</artifactId>
+        </dependency>
```

Comment:
remove the next dependencies:
`spring-boot-starter-data-jdbc`
`spring-boot-starter-jdbc`
`spring-boot-starter-websocket`

Move this `spring-boot-starter-data-jpa`  dependency to other spring-boot dependencies to keep them all in one group
`spring-boot-devtools`